app/controllers/user.js

```diff
@@ -0,0 +1,39 @@
+const { validationResult } = require('express-validator/check');
+const bcrypt = require('bcryptjs');
+const salt = bcrypt.genSaltSync(10);
```

Comment:
Encryption should be handled by a service or utils, also 10 should be a constant or environment variable